bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -239,11 +238,21 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
     </div>
     <p><a href="#top">{{_('Back to Top')}}</a></p>
 </section>
+
+<!-- dislayed to nonFx users -->
+<div class="not-supported">
+  <header id="main-feature">
+      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
+  </header>
+
+  <section>
+    <p>{{_('Currently the plugin check service is only available to Firefox users.')}}</p>
```

Comment:
Would not be ideal without the string but also not a train smash I believe. I am also guessing languages such as German, French etc. will filter in fast so, it should be ok to go live without the string. Let me ask Jen.